dev/docker-compose.dev.yml

```diff
@@ -16,6 +16,7 @@ services:
             - ../migrations:/app/migrations
             - ../tests:/app/tests
             - ../cli:/app/cli
+            - ../docs:/app/docs
```

Comment:
I would prefer a different folder name instead of "docs" (not only for the docker volume, but for the real folder name). The folder began as a doc folder in the main repo, but this was before the `models.yml` was introduced. Now, all the files that lie there a no longer any kind of "documentation", but instead relevant parts of the setup/code, so we should choose a better name. Maybe something like `data` or `assets`? Although I'm not entirely happy with them, either. Maybe you have a better idea (or someone else)?

Reply:
moved to global/meta for the *.yml and global/data for the 2 json-files.